src/queue.py

```diff
@@ -17,18 +17,17 @@ def enqueue(self, value):
         """Add value to the top of  the queue."""
         return self._dlist.push(value)
 
-    def dequeue(self, value):
+    def dequeue(self):
         """From the bottom of the queue, remove and return value."""
-        self._dlist.shift(value)
-        return self
+        return self._dlist.shift()
 
-    def peek(self, value):
+    def peek(self):
```

Comment:
Again we are just peeking at the top value.  We do not need to pass the peak function any arguments.